src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/index.jsx

```diff
@@ -42,6 +42,7 @@ class OrganizationDeveloperSettings extends AsyncView {
         organization={organization}
         onRemoveApp={this.removeApp}
         showInstallationStatus={false}
+        showAppDashboardLink={app.status === 'published'}
```

Comment:
We can explicitly set this to `false` for now as a way of hiding this feature until it's done. That way we can deploy stuff as we work out the details.